types/devexpress-aspnetcore-bootstrap/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Since this is a new definition, please make sure that all the rules are enabled. the package needs to be created using ```dts-gen --dt```

Reply:
I've regenerated the type definitions, please review it again.